clean_project/clean_file.py

```diff
@@ -0,0 +1,38 @@
+"""
+Analyze and clean up files about all useless punctuation
+"""
+import argparse
+import codecs
+import re
+
+
+def clean_up(src_path, dest_path):
+    """
+    Open a file, clean up and write it in another new file
+    :param src_path: source path
+    :param dest_path: destination path of new file
+    """
+    with codecs.open(src_path, 'r', 'utf-8') as source_file:
+        with codecs.open(dest_path, 'w', 'utf-8') as dest_file:
+            for line in source_file:
+                dest_file.write(sanitize(line))
+
+
+def sanitize(line):
+    """
+    Support the clean up operation deleting all the punctuation except characters: - ' _ @ .
```

Comment:
I'd not leave the fullstop (.) in the files. The other characters are OK as they are (usually) part of a word.